kuka_lbr_iiwa_support/package.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<package>
+  <name>kuka_lbr_iiwa_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the KUKA LBR IIWA (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for KUKA LBR IIWA manipulators. This currently includes the base model.
+    </p>
+    <p>
+      Joint limits and max joint velocities are based on the information in
+      the KUKA data sheets online. 
+      http://www.kuka-robotics.com/res/sps/e6c77545-9030-49b1-93f5-4d17c92173aa_Spez_LBR_iiwa_en.pdf
+      All urdf's / xacro's are based on the default motion and joint velocity 
+      limits, unless noted otherwise (ie: no support for high speed joints, 
+      extended / limited motion ranges or other options).
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+  <author email="levi.armstrong@swri.org">Levi Armstrong (Southwest Research Institute)</author>
+  <maintainer email="levi.armstrong@swri.org">Levi Armstrong (Southwest Research Institute)</maintainer>
+
+  <license>Apache 2.0</license>
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>roslaunch</build_depend>
+
+  <run_depend>joint_state_publisher</run_depend>
+  <run_depend>robot_state_publisher</run_depend>
+  <run_depend>rviz</run_depend>
+</package>
```

Comment:
This seems to be missing `url` elements for `website`, `bugtracker` and `repository`.